api/server.js

```diff
@@ -29,8 +29,19 @@ app.use(auth({
 /* static routes */
 app.use(express.static(path.join(__dirname, 'build')));
 
-const rawData = fs.readFileSync('ingredients.json');
-const ingredients = JSON.parse(rawData);
+const rawDataIngredients = fs.readFileSync('./Resources/ingredients.json', 'utf8');
```

Comment:
since this has grown so much let's move all this and the route to a separate router and place it in the Resources directory. BTW please rename the directory "Resources" -> "resources".